templates/download/desktop/index.html

```diff
@@ -34,20 +39,24 @@ <h2>Ubuntu {{ releases.lts.full_version }} LTS</h2>
     <div class="col-4">
       <div class="u-align--center u-sv4">
         {{ image (
-          url="https://assets.ubuntu.com/v1/b743f1ee-jj-in-circle.png",
-          alt="",
-          width="200",
-          height="200",
-          hi_def=True,
-          loading="auto"
-          ) | safe
+        url="https://assets.ubuntu.com/v1/b743f1ee-jj-in-circle.png",
+        alt="",
+        width="200",
+        height="200",
+        hi_def=True,
+        loading="auto"
+        ) | safe
```

Comment:
This is why I don't use prettier for Jinja, sometimes it may even break it. The proper indentation of Jinja here should be

```Jinja
{{
  image (
    url="https://assets.ubuntu.com/v1/b743f1ee-jj-in-circle.png",
    alt="",
    width="200",
    height="200",
    hi_def=True,
    loading="auto"
  ) | safe
}}
```